schema.graphql

```diff
@@ -1,6 +1,6 @@
 type Collection @entity {
   id: ID!
-  collectionId: String! @index
+  collectionId: Float! @index
```

Comment:
Why not `ID` instead of `Float`?

Reply:
ID should be unique and we are storing destroyed collections and burnt items to show NFT transfers (past)

Reply:
So then how about `Int`?